academic-requirements/src/components/FourYearPlanPage.tsx

```diff
@@ -8,57 +15,120 @@ const FourYearPlanPage = (props: {
     number: number;
     semesters: string;
     subject: string;
+    category: string;
   }[];
   concentrationCourseList: {
     credits: number;
     name: string;
     number: number;
     semesters: string;
     subject: string;
+    category: string;
   }[];
+  onClickCategory(category: string): void; //Hovland 7Nov22
+
 }) => {
+  //Stuff for category dropdown. Hovland 7Nov22
+  const [category, setCategory] = useState(""); //category that is selected
+  const [categories, setCategories] = useState<string[]>([]);
   //Functions and variables for controlling an error popup
   const [visibility, setVisibility] = useState(false);
   const popupCloseHandler = () => {
     setVisibility(false);
   };
   const [error, setError] = useState("");
+
+  useEffect(() => {
+    console.log("running")
```

Comment:
Minor nitpick but this console.log isn't needed

Reply:
See this comment as well: https://github.com/CS-458/Academic_Requirements/pull/22#issuecomment-1320809056